outputs/openapi/components.go

```diff
@@ -170,7 +173,15 @@ func componentToSchema(service *astra.Service, component astra.Field, bindingTyp
 			if !fieldBinding.NotShown {
 				fieldSchema, fieldBound := componentToSchema(service, field, bindingType)
 				if fieldBound {
+					_ = mergo.Merge(&fieldSchema, field.StructFieldValidationTags[astTraversal.GinValidationTag])
```

Comment:
What's the response here? An error? Why aren't we handling it if it is?

Reply:
I have now updated it to handle and log the error.

Reply:
Does the function not return an error? Is it worth allowing it to return an error?